js/apps/admin-ui/src/i18n/i18n.ts

```diff
@@ -18,7 +18,7 @@ export const i18n = createInstance({
   },
   backend: {
     loadPath: joinPath(
-      environment.authServerUrl,
+      environment.adminBaseUrl,
```

Comment:
This will now always use the configured Admin URL to get the localizations, rather than the regular Keycloak server root. I am not entirely sure if this will work correctly in all configurations, but from what I have tested this works alright locally.